src/pages/FindID/FindID.scss

```
@@ -0,0 +1,42 @@
+.idFindFrame {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  .idText {
+    font-size: 25px;
+    font-weight: 600;
+    color: rgb(166, 166, 166);
+  }
+  .findInputFrame {
+    display: flex;
+    flex-direction: column;
+    margin-top: 30px;
+    .nameInput,
+    .phoneInput {
+      width: 370px;
+      padding: 15px;
+      border: none;
+      border-bottom: 1px solid gray;
+      margin-bottom: 35px;
+    }
+  }
```

선택자와 선택자 사이에 한 줄 여백을 줘서 가독성을 신경써주세요
```suggestion
  .findInputFrame {
    display: flex;
    flex-direction: column;
    margin-top: 30px;
    
    .nameInput,
    .phoneInput {
      width: 370px;
      padding: 15px;
      border: none;
      border-bottom: 1px solid gray;
      margin-bottom: 35px;
    }
  }
```

---

넵 반영 완료 하였습니다!